### PR TITLE
Fix gateway templates

### PR DESCRIPTION
**Pull request by amarthadan**
[Issue AN-603](https://api3dao.atlassian.net/browse/AN-603)

### CODE REVIEW

File: packages/airnode-deployer/terraform/airnode/aws/templates/httpSignedDataApiGateway.yaml.tpl [hunk 1]
```diff
@@ -35,10 +35,7 @@ components:
 
     EndpointRequestExample:
       summary: Endpoint request example
-      value: { "parameters": { "from": "EUR", "amount": 5 } }
-    EndpointResponseExample:
```
Comment (andreogle) [inline packages/airnode-deployer/terraform/airnode/aws/templates/httpSignedDataApiGateway.yaml.tpl:39]: Why was the response example removed? Just curious

File: packages/airnode-deployer/terraform/airnode/aws/templates/httpSignedDataApiGateway.yaml.tpl [hunk 2]
```diff
@@ -35,10 +35,7 @@ components:
...
-      summary: Endpoint response example
-      value: { "response": 20 }
+      value: { "encodedParameters": "0x31737300000000000000000000...0000000000000000000000" }
```
Comment (andreogle) [inline packages/airnode-deployer/terraform/airnode/aws/templates/httpSignedDataApiGateway.yaml.tpl:38]: It doesn't matter that this value has been truncated, right?

**Review by andreogle — APPROVED** [review]

File: packages/airnode-deployer/terraform/airnode/aws/templates/httpSignedDataApiGateway.yaml.tpl (hunk 1 above)
Comment (amarthadan) [reply packages/airnode-deployer/terraform/airnode/aws/templates/httpSignedDataApiGateway.yaml.tpl:39]: Just out of convenience. We don't have these in the GCP version (they're using old swagger format so we can't), it's constantly outdated and it doesn't show anywhere in AWS.

File: packages/airnode-deployer/terraform/airnode/aws/templates/httpSignedDataApiGateway.yaml.tpl (hunk 2 above)
Comment (amarthadan) [reply packages/airnode-deployer/terraform/airnode/aws/templates/httpSignedDataApiGateway.yaml.tpl:38]: No, it's just an example.

**Pull request merged**